hawkbit-dmf-parent/hawkbit-dmf-rabbitmq-test/src/main/java/org/eclipse/hawkbit/rabbitmq/test/RabbitMqSetupService.java

```diff
@@ -30,7 +30,8 @@
  */
 public class RabbitMqSetupService {
 
-    private static Client rabbitmqHttpClient;
+    private static final String GUEST = "guest";
```

Comment:
Better split this into two variables: `DEFAULT_USER` and `DEFAULT_PASSWORD`. Than code gets more readable.